test/intl402/NumberFormat/prototype/formatRangeToParts/argument-is-infinity-throws.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.NumberFormat-formatRangeToParts
+description: Throws a RangeError exception if start or end argument is cast to an Infinity value.
+locale: [en-US]
+features: [Intl.NumberFormat-formatRangeToParts]
+---*/
+
+const nf = new Intl.NumberFormat("en-US", {signDisplay: "exceptZero"});
+
+// 5. If x or y is a non-finite Number ..., throw a RangeError exception.
```

Comment:
Again, this requirement does not exist.  x and y are allowed to be infinite.  I don't know where you got this, unless you are looking at a much older version of the spec.  This requirement existed at Stage 2 but was removed before Stage 3.

Reply:
Confusion between actual chromium patch and spec, being more strict with the spec, updates in progress